packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountSearchBox.tsx

```diff
@@ -13,6 +13,7 @@ const StyledInput = styled(Input)`
     input {
         /* to line up with the coin filter  */
         padding-left: 46px;
+        min-height: 38px;
```

Comment:
Could you tell me what is this here for? Cannot understand :(

Reply:
The plus icon next to this input is 38px high so wanted to aligned the input with the plus icon.